pkg/controller/node/nodecontroller.go

```diff
@@ -125,15 +126,16 @@ type NodeController struct {
 	// it doesn't receive update for this amount of time, it will start posting "NodeReady==
 	// ConditionUnknown". The amount of time before which NodeController start evicting pods
 	// is controlled via flag 'pod-eviction-timeout'.
-	// Note: be cautious when changing the constant, it must work with nodeStatusUpdateFrequency
+	// Note: be cautious when changing the constant, it must work with NodeStatusUpdateFrequency
+	// which should be shared and common between all kubelets.
 	// in kubelet. There are several constraints:
-	// 1. nodeMonitorGracePeriod must be N times more than nodeStatusUpdateFrequency, where
-	//    N means number of retries allowed for kubelet to post node status. It is pointless
-	//    to make nodeMonitorGracePeriod be less than nodeStatusUpdateFrequency, since there
-	//    will only be fresh values from Kubelet at an interval of nodeStatusUpdateFrequency.
-	//    The constant must be less than podEvictionTimeout.
-	// 2. nodeMonitorGracePeriod can't be too large for user experience - larger value takes
-	//    longer for user to see up-to-date node status.
+	// 1. nodeMonitorGracePeriod MUST be >= nodeStatusUpdateRetry * nodeStatusUpdateFrequency.
```

Comment:
Does nodeStatusUpdateRetry need to start with a capital letter? I assume you're referring to the exported constant?